### DISCUSSION TITLE
User data collected for Privacy Policy

### DISCUSSION THREAD

**Discussion by bluecaret** [Glench/ExtPay#43] [Q&A] [answered]
Hello, I'm trying to figure out what to do for a privacy policy for my extension. If the only info collected from my extension is what ExtensionPay requires, could someone list what that info is, and if it would even count to include in my Privacy Policy? I'm really not sure how to approach this. I'm looking at using this policy generator if it helps for the info I need: https://app.privacypolicies.com/wizard/privacy-policy

Thanks!

**Comment by burkybang** [comment, marked as answer]
ExtensionPay's privacy policy is listed in the FAQ at the bottom of https://extensionpay.com.

In the privacy policy for my extension, I explained where to find that.

**Comment by karlosos** [comment]
Could you provide a sample privacy policy @burkybang?

**Reply by burkybang** [reply]
https://docs.google.com/document/d/e/2PACX-1vTnS915LjPNLpMOl1WIKhoFqjAb53c0F8kXnYzDgpfkBhzQaVBWzZBwrLMab_JBR_z70t9wP7fpnDIK/pub
